README.rst

```diff
@@ -24,6 +26,17 @@ It includes a remote control layer to control boards connected to other hosts.
 
   * `Install Development State <#install-development-state>`_
 
+
+Getting started
+---------------
+
+There is a tutorial series on the pengutronix youtube channel you can follow to
+get started: `Labgrid Tutorial Playlist
+<https://www.youtube.com/playlist?list=PLPy6BX4pJosCEq7CIU06bt2WM7lFAy1CF>`_.
+
+The other starting point is in the `documentation
+<https://labgrid.readthedocs.io/en/v24.0.1/getting_started.html>`_.
```

Comment:
```suggestion
<https://labgrid.readthedocs.io/en/latest/getting_started.html>`_.
```

Reply:
I intentionally linked to the latest release tag to let it somewhat match the youtube tutorials which I exptected to use crossbar. But looking at the tutorials again there is no remote usage, i'll update.